src/evo/mnhftx.cpp

```diff
@@ -308,7 +305,6 @@ void CMNHFManager::AddToCache(const Signals& signals, const CBlockIndex* const p
     const uint256& blockHash = pindex->GetBlockHash();
     {
         LOCK(cs_cache);
-        LogPrintf("CMNHFManager::AddToCache: mnhf for block %s add to cache: %lld\n", pindex->GetBlockHash().ToString(), signals.size());
```

Comment:
same, `if (!signals.empty()) { ... }`

Reply:
I dont quite agree, because since MN_RR signalled once and till the moment it is buried in next release it would be always this log record for each block because `signals` would have exactly one record

Reply:
hmm. ok 👍